integration_test/run.sh

```diff
@@ -7,9 +7,7 @@ rm -rf ${TESTDIR}
 mkdir -p ${TESTDIR}/1
 
 # To kill any remaining open bitcoind.
-killall -9 bitcoind
-
-echo $PATH
+killall -9 bitcoind 2> /dev/null
```

Comment:
lol, I had no idea this line was here. This is pretty anti-social. But it has nothing to do with this PR, so that's fine.

Reply:
Couldn't think of a quick thorough fix so I raised an issue: https://github.com/rust-bitcoin/rust-miniscript/issues/395

Reply:
I may be to blame here. I was trying to make the script work without thinking of all the consequences.